src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js

```diff
@@ -173,7 +178,11 @@ const ValidatorsMonitor = ({ watchList }) => {
                 <SecondaryButton>{t('Stakes')}</SecondaryButton>
               </Link>
             )}
-            <ValidatorActionButton isValidator={isValidator} address={address} />
+            <ValidatorActionButton
+              isValidator={isValidator}
+              isLoadingValidators={isLoadingValidators}
+              address={address}
```

Comment:
`ValidatorActionButton` takes only two props. Therefore `isLoadingValidators` can either be added to its definition or used differently here

Reply:
This prop was not necessary as its handled via isValidator, so removed.